Failed to retrieve Microsoft Defender status: Process terminated with unexpected result: Not running

<!-- Thank you for reporting this issue! -->
<!-- The following template is highly recommended to trigger an efficient course of actions: it's pretty suitable if you're not familiar with the project as it will guide you in sharing the most important information. If you're not comfortable with this template for your particular report, feel free to erase it and write your report from scratch -->

Let's make sure issue is not already fixed in latest builds first.

- [X] I verified I can reproduce this issue against [latest Integration Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/)

## Steps to reproduce

Disable Windows defender completely, e.g. using https://www.sordum.org/9480/defender-control-v2-1/

`powershell -Command (Get-MpComputerStatus).AMRunningMode` then returns an additional status not documented by Microsoft: `Not running`.

https://github.com/eclipse-platform/eclipse.platform.ui/blob/54849021b21ba173e6008f44d294423017ea456a/bundles/org.eclipse.ui.workbench/Eclipse%20UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java#L325-L328

should be adjusted to include that.

I wonder: Maybe this method should be more graceful to unknown statuses, e.g. log them and return `false` instead of throwing an Exception.

## Here is some relevant log output

From `<workspace>/.metadata/.log`
```
org.eclipse.core.runtime.CoreException: Failed to retrieve Microsoft Defender status.
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.isWindowsDefenderActive(WindowsDefenderConfigurator.java:331)
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.runExclusionCheck(WindowsDefenderConfigurator.java:159)
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.lambda$0(WindowsDefenderConfigurator.java:106)
	at org.eclipse.core.runtime.jobs.Job$2.run(Job.java:187)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.io.IOException: Process terminated with unexpected result:
Not running
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.isWindowsDefenderActive(WindowsDefenderConfigurator.java:328)
	... 4 more
```

## Tested under this environment:
* OS & version: Windows 11 Build 27729

--- a/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java
+++ b/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java
@@ -323,7 +323,8 @@ public class WindowsDefenderConfigurator implements EventHandler {
 			return switch (onlyLine.toLowerCase(Locale.ENGLISH).strip()) {
 			// Known values as listed in
 			// https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows#use-powershell-to-check-the-status-of-microsoft-defender-antivirus
-			case "sxs passive mode", "passive mode", "" -> false; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+			// "not running" status appears to be undocumented (https://github.com/eclipse-platform/eclipse.platform.ui/issues/2447)
+			case "sxs passive mode", "passive mode", "not running", "" -> false; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$
 			case "normal", "edr block mode" -> true; //$NON-NLS-1$//$NON-NLS-2$
 			default -> throw new IOException("Process terminated with unexpected result:\n" + String.join("\n", lines)); //$NON-NLS-1$//$NON-NLS-2$
 			};

